modules/sno-du-conn-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-connected.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-conn-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+A key feature to enable as part of a single node installation is workload partitioning. This limits the cores allowed to run platform services, maximizing the CPU core for application payloads. You must configure workload partitioning at cluster installation time.
+
+[NOTE]
+====
+Workload partitioning must be applied during installation.
+====
+
+.Procedure
+
+* The base64-encoded content below contains the CPU set that the management workloads are constrained to.
+This content must be adjusted to match the reserved cores defined in the `performanceprofile` and must be accurate for
+the number of cores on the cluster.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 02-master-workload-partitioning
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZS53b3JrbG9hZHMubWFuYWdlbWVudF0KYWN0aXZhdGlvbl9hbm5vdGF0aW9uID0gInRhcmdldC53b3JrbG9hZC5vcGVuc2hpZnQuaW8vbWFuYWdlbWVudCIKYW5ub3RhdGlvbl9wcmVmaXggPSAicmVzb3VyY2VzLndvcmtsb2FkLm9wZW5zaGlmdC5pbyIKcmVzb3VyY2VzID0geyAiY3B1c2hhcmVzIiA9IDAsICJjcHVzZXQiID0gIjAtMSw1Mi01MyIgfQo=
```

Comment:
We should have an note here to explain how to compose this data, because this is different based on hardware, and it has to match what is configured in crio, and then config performanceprofile and tuned has to also match. 

e.g., currently after decoding this looks like the following, which does not align with performanceprofile config in the same doc:

```
[crio.runtime.workloads.management]
activation_annotation = "target.workload.openshift.io/management"
annotation_prefix = "resources.workload.openshift.io"
resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
```